### PR TITLE
Overhaul forms for pda and worksites to be nested and more awesome

### PR DESCRIPTION
**Pull request by tabiodun**

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -13,6 +13,8 @@ on:
       - '**/**.js.snap'
       - '.github/workflows/*'
   push:
+    tags:
+      - 'v*'
```
Comment (BradenM) [inline .github/workflows/main.yml]: @tabiodun ahh! this snuck back in here!

Comment (tabiodun) [reply .github/workflows/main.yml]: Just removed it lol

Comment (BradenM) [reply .github/workflows/main.yml]: haha, for real though this PR is awesome :100:

Comment (tabiodun) [reply .github/workflows/main.yml]: 👍

**Pull request merged**